scripts/loki/lokistack.yaml

```
@@ -0,0 +1,36 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: lokistack-template
+objects:
+  - apiVersion: loki.grafana.com/v1
+    kind: LokiStack
+    metadata:
+      name: lokistack
+      namespace: netobserv
+    spec:
+      limits:
+        global:
+          ingestion:
+            ingestionBurstSize: 100
+            ingestionRate: 500
+            maxGlobalStreamsPerTenant: 100000
+            perStreamRateLimit: 9
+            perStreamRateLimitBurst: 45
+      tenants:
+        mode: openshift-network
+      managementState: Managed
+      storage:
+        schemas:
+          - effectiveDate: '2023-10-15'
+            version: v13
+        secret:
+          name: s3-secret
+          type: s3
+      size: ${SIZE}
+      storageClassName: ${DEFAULT_SC}
+parameters:
+  - name: SIZE
+    value: "1x.extra-small"
+  - name: DEFAULT_SC
+    value: "gp2"
```

```suggestion
    value: "gp3-csi"
```
I don't see gp2 as SC:
```
$ oc get sc
NAME                PROVISIONER       RECLAIMPOLICY   VOLUMEBINDINGMODE      ALLOWVOLUMEEXPANSION   AGE
gp2-csi             ebs.csi.aws.com   Delete          WaitForFirstConsumer   true                   4h16m
gp3-csi (default)   ebs.csi.aws.com   Delete          WaitForFirstConsumer   true                   4h16m
```

---

Thats the default we use in our backend tests [lokistack-simple.yaml#L38](https://github.com/openshift/openshift-tests-private/blob/master/test/extended/testdata/netobserv/loki/lokistack-simple.yaml#L38) so used the same.
Will update it in both places